File: Desarrollos AddOn SAP B1/Samples/COM UI/CSharp/22.RightClick/SubMain.cs
//   SAP MANAGE UI API 2007 SDK Sample
// ****************************************************************************
// 
//   File:      SubMain.vb
// 
//   Copyright (c) SAP MANAGE
// 
//  THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF
//  ANY KIND, EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO
//  THE IMPLIED WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A
//  PARTICULAR PURPOSE.
// 
// ****************************************************************************

using System;
using System.Windows.Forms;

class SubMain { 
    
    public static void Main() { 
        
        //  Creating an object
        RightClick oRightClick = null; 
        
        oRightClick = new RightClick(); 
        
        //  Starting the Application
        System.Windows.Forms.Application.Run(); 
    }   
} 

File: Desarrollos AddOn SAP B1/Samples/COM UI/VB.NET/16.ActiveX/SubMain.vb
Option Strict Off
Option Explicit On
Module SubMain
    '//  SAP MANAGE UIDI API 2006 SDK Sample
	'//****************************************************************************
	'//
	'//  File:      SubMain.bas
	'//
	'//  Copyright (c) SAP MANAGE
	'//
	'// THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF
	'// ANY KIND, EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO
	'// THE IMPLIED WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A
	'// PARTICULAR PURPOSE.
	'//
	'//****************************************************************************
	
	
	Public Sub Main()
		
        Dim oActiveXTree As ActiveXTree
		
        oActiveXTree = New ActiveXTree

        Windows.Forms.Application.Run()
	End Sub
End Module